### DISCUSSION TITLE
Billing_Code_Type_Version Question

### DISCUSSION THREAD

**Discussion by angelamarie0975** [CMSgov/price-transparency-guide#314] [General]
Adding a new discussion because the thread I am asking about is marked as answered, so I am not sure my response will get addressed.

@shaselton-usds - in response to your last comment on https://github.com/CMSgov/price-transparency-guide/discussions/203 - I am just trying to make sure that I fully understand your response. In your response you stated that if we don't have versioning for codes 'like you would find with ICD, DRG, or CPT', using the plan year instead is ok, but that can be interpreted in 2 ways:

- If organization has codes (not ICD, DRG, or CPT) but no versioning stored in their system - ok to use plan year for those codes, but for ICD, DRG, or CPT codes, get the versioning from CMS/CPT
- If organization has no versioning stored for ANY billing code type, ok to use plan year instead regardless of bill code type

If we do not store the version for DRG codes in our systems, are we expected to apply the versioning for DRG codes from the files that are published publicly by CMS? If so, that means that we will need to create more logic to pull in this data to store it for MRF purposes only since we don't house that data in our system. Just want to make sure we know exactly what the expectation is.

**Comment by shaselton-usds** [comment]
Just making sure that I understand the situation: a company may have negotiations for specific billing code types (that have versions -- DRG, CPT, etc) and the respective billing codes, but don't what version of the billing type that the negotiation is for? 

What would be the typical assumption of the version for a billing code type that has a negotiated rate be? Whatever the latest version is?

**Reply by angelamarie0975** [reply]
In our situation, we are pulling the data from our claims systems as we store the billing code data there. However, there was not ever a requirement to store the versions in our tables for claims payment reasons, so we don't have it readily available. Basically, our rates are stored by the billing codes - so if a billing code is no longer effective, this is normally identified by effective/term dates in our claims payment system, so we don't really require the actual version - at least that is my understanding of it. We know what the versions are and can get that data if necessary, but if it's not truly required in the MRF and we can use the CCYY instead - that would be way more cost effective for us to produce that data value than to add in extra logic to get the actual version value. I am not sure if other organizations store the versions in their system - but based on the 203 discussion, it sounded like others do not have this information either. 

So in short - I guess the answer would be yes - whatever the latest version would be. We do not have our contracts based on specific billing code versions - they are set up per the active codes and if the codes are no longer active, we won't be reporting them in the MRF as they should no longer be attached to any contracts.
